java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogTransactionStack.java

```diff
@@ -79,7 +81,7 @@ private void createTransactionDeletionLambda(InstanceProperties instanceProperti
                 .runtime(Runtime.JAVA_11)
                 .handler("sleeper.statestore.transaction.TransactionLogTransactionDeletionTriggerLambda::handleRequest")
                 .environment(Utils.createDefaultEnvironment(instanceProperties))
-                .reservedConcurrentExecutions(1)
+                .reservedConcurrentExecutions(instanceProperties.getInt(TRANSACTION_LOG_TRANSACTION_DELETION_LAMBDA_CONCURRENCY_RESERVED))
```

Comment:
This should be set on the deletion function. The trigger function needs this set to 1.

Reply:
Addressed